cli_tools/common/disk/inspect.go

```diff
@@ -38,8 +38,18 @@ type Inspector interface {
 
 // InspectionResult contains the partition and boot-related properties of a disk.
 type InspectionResult struct {
-	// HasEFIPartition indicates whether the disk has a EFI partition.
-	HasEFIPartition                    bool
+	// UEFIBootable indicates whether the disk is bootable with UEFI.
```

Comment:
This field should indicate whether the identified OS is bootable with UEFI. The next field is similar, but should indicate whether it's bootable with BIOS.

The current docs talk about the *disk*.

Reply:
If that's the case, then shouldn't the variable be: "HasEFIPartition" or something like that?

Reply:
done. for "hybrid mbr" we exclude it; for "protectvie mbr" we exclude if "BIOS boot" partition exists.